.gitlab-ci.yml

```diff
@@ -994,8 +1003,8 @@ test_full_integration:
         integration/extra/release_tool.py --set-version-of $repo --version pr;
       done
     # Other dependencies
-    - tar -xf stage-artifacts/host-tools.tar ./mender-artifact && mv mender-artifact /usr/local/bin/
-    - tar -xf stage-artifacts/host-tools.tar ./directory-artifact-gen && mv directory-artifact-gen /usr/local/bin/
+    - install stage-artifacts/mender-artifact-linux /usr/local/bin/mender-artifact
+    - install ${WORKSPACE}/go/src/github.com/mendersoftware/mender/support/modules-artifact-gen/directory-artifact-gen /usr/local/bin/
```

Comment:
Not very important, but for the future, you could consider doing `make -C ${WORKSPACE}/go/src/github.com/mendersoftware/mender install-modules-gen` to get them all.

Reply:
Thanks for a good tip!